api_v3/lib/types/filters/advanced/KalturaSearchComparableAttributeCondition.php

```diff
@@ -43,4 +43,10 @@ public function toObject($objectToFill = null, $propsToSkip = array())
 		$objectToFill->setField($field);
 		return $objectToFill;
 	}
+
+	protected function doFromObject($srcObj, KalturaDetachedResponseProfile $responseProfile = null)
+	{
+		if ($srcObj instanceof AdvancedSearchFilterComparableAttributeCondition)
```

Comment:
1. I think you can assume that $srcObj is AdvancedSearchFilterComparableAttributeCondition.
2. Please check if $this->shouldGet('attribute', $responseProfile).
3. Same for KalturaSearchMatchAttributeCondition.